packages/template-retail-react-app/app/translations/README.md

```diff
@@ -74,13 +74,13 @@ The process for adding a new locale is as follows:
 
 1. Create/enable the new locale in Business Manager of your B2C Commerce instance
 2. Enable the locale's currency too in Business Manager
-3. Add the new locale and its currency to `pwa/package.json`
+3. Add the new locale and its currency to `package.json`
 4. If the new locale is your new default locale:
     - Update your React components' default messages to the new default locale's language
-    - In `pwa/package.json`, set the new default locale
-    - Run `npm run extract-default-translations` to extract the new default locale's translations into `pwa/app/translations/[defaultLocale].json`
+    - In `package.json`, set the new default locale
+    - Run `npm run extract-default-translations` to extract the new default locale's translations into `app/translations/[defaultLocale].json`
     - Send the extracted translations to your translation team
-5. Place the files you receive from your translation team into the `pwa/app/translations/` folder
+5. Place the files you receive from your translation team into the `app/translations/` folder
```

Comment:
This will also need updating to reflect the current state of the retail-react-app and it's configuration of locale. @vmarta or @alexvuong would be the right resource to help fix that up.

Reply:
Sure, let me update the doc. @wjhsf I am gonna push the changes to this branch if you would not mind.